app/mailers/cards_mailer.rb

```diff
@@ -1,4 +1,5 @@
 class CardsMailer < ActionMailer::Base
+  layout 'mailers'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.